Tests/Routing/ChainRouterTest.php

```diff
@@ -680,3 +710,7 @@ protected function createRouterMocks()
 abstract class RequestMatcher implements \Symfony\Component\Routing\RouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
 {
 }
+
+abstract class VersatileRouter implements \Symfony\Component\Routing\RouterInterface, \Symfony\Cmf\Component\Routing\ChainedRouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
```

Comment:
travis is really unhappy about this one. do we need it? i think you did not have this in the previous pull request.